src/vs/workbench/contrib/chat/browser/chatListRenderer.ts

```diff
@@ -488,6 +488,9 @@ export class ChatListItemRenderer extends Disposable implements ITreeRenderer<Ch
 				parts.push(newPart);
 			}
 		});
+		if (templateData.renderedParts) {
```

Comment:
Good catch. I was just wondering why I still do this totally differently in this case rather than do the same 'diff' flow as in the progressive rendering case. I should just make the two cases exactly the same.